custom-recipes/data_queries/recipe.py

```diff
@@ -126,7 +127,19 @@
         
         if 'item_schema' not in f.keys() or len(f['item_schema']) == 0:
             continue
-        
+
+        # if user set his list of fields
+        if 'fields' in json.loads(config['oql']).keys():
+            user_fields = json.loads(config['oql'])['fields']
+
+            # display error in logs if field does not exist
+            print([f'bad field: {item}' for item in user_fields if item not in f['dataset_schema_field_list']])
```

Comment:
Hi @Marsobad,

I wrote the check you asked.
I've prefered raise an error instead of parsing string as a list for 2 reasons :
- educate users because there will be no such trick when he will use directly our API
- in a string users might have separate fields with different separators, I cannot try all possibilities and if the string is badly parsed, users will have to wait the end of the job to see the mistake and it could take hours.

I'm going on documentation update and send it to you soon.
Thanks